projects/atomic-swap-service/headless-wallet.md

```diff
@@ -79,25 +85,28 @@ This will allow the wallet user to add any existing proposal to the wallet's `li
 
 To ensure the informed parameters are correct, a `get` request will also be made at this time to the service backend, and a validation executed on the received contents. Any failure found on the process will be thrown and the registration discarded.
 
-### Fetch
+### Listened Proposals endpoint
+A new route `[GET] /wallet/atomic-swap/tx-proposal/listenedProposals` will be created, allowing the user to retrieve the list of `proposalIds` currently being listened to. No password should be retrieved on this call.
```

Comment:
Maybe `/wallet/atomic-swap/tx-proposal/list` ?
What do you think?

Reply:
It's a good suggestion: renamed it.